instrumentation/github.com/go-sql-driver/mysql/splunkmysql/test/integration_test.go

```diff
@@ -0,0 +1,219 @@
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build linux
+// +build linux
```

Comment:
It would be better if is filtered on the GitHub Actions workflow level.

Reply:
Created issue to track it https://github.com/signalfx/splunk-otel-go/issues/122

Reply:
Yeah, I'm all for this! I didn't know how to construct a `goyek` command to do that. Agreed that we should resolve this with #122.